Sources/NIOSSH/Child Channels/SSHChannelMultiplexer.swift

```diff
@@ -93,6 +93,9 @@ extension SSHChannelMultiplexer {
             self.erroredChannels.append(channelID)
         }
     }
+    
+    // The username which the server accepted in authorization
+    var username : String? { (delegate as? NIOSSHHandler)?.username }
```

Comment:
Rather than cast the delegate, can we add the username computed property to it?

Also, while we're here, a quick NIO style nit:

```suggestion
    var username: String? { (delegate as? NIOSSHHandler)?.username }
```